README.md

```diff
@@ -68,34 +71,38 @@ This repository's contents is divided across four primary sections:
 - `/packages` contains the source for our public packages
 - `/scripts` contains scripts used for building and publishing
 - `/assets` contains icons and translations relied on by the app
+- `/docs` contains the content for our docs site at [tldraw.dev](https://tldraw.dev)
 
 ### Applications
 
 - `examples`: our local development / examples project
-- `docs`: our docs site at [tldraw.dev](https://tldraw.dev)
 - `vscode`: our [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=tldraw-org.tldraw-vscode)
 
 ### Packages
 
+- `assets`: a library for working with tldraw's fonts and translations
 - `editor`: the tldraw editor
-- `ui`: the editor's user interface
-- `tldraw`: the main tldraw package containing both the editor and the UI
+- `file-format`: a library for working with tldraw's `.tldr` file format
+- `indices`: a library for working with tldraw's indices
+- `polyfills`: a collection of polyfills used by tldraw
```

Comment:
Do we really want to mention all these packages here? Should we reduce the list to just the main ones? We could always move some of the internal ones to the contributing guide. I'm good either way, just a thought.